utils/imodels-client-test-utils/src/assertions/NodeOnlyAssertions.ts

```diff
@@ -170,6 +170,7 @@ export async function assertChangesetGroup(params: {
 export function assertLock(params: {
   actualLock: Lock;
   expectedLock: Lock;
+  lockLevel?: LockLevel;
```

Comment:
why does this have to be optional?

Reply:
Took a look at it again and it's not actually needed since we can assert it from expectedLock. Thanks